src/private/menus.ts

```diff
@@ -26,6 +26,23 @@ export namespace menus {
      */
     contentDescription?: string;
   }
+
+  /**
+   * Defines how a menu item should appear in the NavBar.
+   */
+  export enum DisplayMode {
```

Comment:
The naming here is a bit confusing. DisplayMode.never sounds like it should not be displayed at all. The fact that overflow is part of the explanation but not in the name is, I think, part of the confusion. How about something like:
default = 0, // show unless there's no room, in which case push it into overflow
overflowOnly = 1, // show in overflow alway

Reply:
Agree, however `never` to `overflowOnly`
However, instead of `default`, I feel it should be still `ifRoom` to indicate that intended behaviour.

Reply:
@ht4963-ms Let me know what you think.

Reply:
I think ifRoom **is** the default behavior because you will never force something to be displayed if there's no room, so this distinction is unnecessary. By specifying "ifRoom", it begs the question: does it not get displayed at all when there's no room, then you'll have add the additional explanation that it does get displayed, but only in the overflow.